crates/bevy-inspector-egui/src/bevy_inspector/mod.rs

```diff
@@ -205,7 +205,7 @@ pub fn ui_for_state<T: States + Reflect>(world: &mut World, ui: &mut egui::Ui) {
     };
     let mut env = InspectorUi::for_bevy(&type_registry, &mut cx);
 
-    let mut current = state.0.clone();
+    let mut current = state.clone();
```

Comment:
To be truly identical it should be `state.get().clone()`, I don't know whether that actually makes a difference here.

Reply:
Updated to `state.get().clone()`.